### PR TITLE
Added MergeRequestFilter.in and MergeRequestFilter.wip

### PR DESCRIPTION
**Pull request by burtbai**
see: <https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests>

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java [hunk 1]
```diff
@@ -36,6 +37,8 @@
     private String sourceBranch;
     private String targetBranch;
     private String search;
+    private String in;
+    private MergeRequestFilterWIP wip;
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java]: ```wip``` would be better as a Boolean and then simply change it to ```"yes"``` or "```"no"``` when used as I have shown below.

File: src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java [hunk 2]
```diff
@@ -284,14 +287,40 @@ public MergeRequestFilter withSearch(String search) {
...
+    }
+
+    public MergeRequestFilter withIn(String in) {
+        this.in = in;
+        return (this);
+    }
+
+    public MergeRequestFilterWIP getWip() {
+        return wip;
+    }
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java]: Should return a ```Boolean```

File: src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java [hunk 3]
```diff
@@ -284,14 +287,40 @@ public MergeRequestFilter withSearch(String search) {
...
+
+    public void setWip(MergeRequestFilterWIP wip) {
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java]: Should be changed to a ```Boolean```

File: src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java [hunk 4]
```diff
@@ -284,14 +287,40 @@ public MergeRequestFilter withSearch(String search) {
...
+        this.wip = wip;
+    }
+
+    public MergeRequestFilter withWip(MergeRequestFilterWIP wip) {
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java]: Should be changed to a Boolean

File: src/main/java/org/gitlab4j/api/Constants.java [hunk 5]
```diff
@@ -350,6 +350,27 @@ public String toString() {
...
+        @JsonValue
+        public String toValue() {
+            return (enumHelper.toString(this));
+        }
+
+        @Override
+        public String toString() {
+            return (enumHelper.toString(this));
+        }
+    }
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/Constants.java]: As stated below this is not needed.

File: src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java [hunk 6]
```diff
@@ -310,6 +339,8 @@ public GitLabApiForm getQueryParams() {
             .withParam("my_reaction_emoji", myReactionEmoji)  
             .withParam("source_branch", sourceBranch)
             .withParam("target_branch", targetBranch)
-            .withParam("search", search));
+            .withParam("search", search)
+            .withParam("in", in)
+            .withParam("wip", wip));
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/MergeRequestFilter.java]: Change this to:
```
.withParam("wip", (wip == null ? null : wip ? "yes" : "no"));
```

**Comment by gmessner** [comment]
@burtbai 
I have merged the PR, will do a release later this week when there are some more updates/fixes to go with it.

**Comment by burtbai** [comment]
@gmessner thank you.

**Pull request merged**